pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "recipe_scrapers"
+description = "Python package, scraping recipes from all over the internet"
+authors = [
+    { name = "Hristo Harsev", email = "r+pypi@hharsev.com" },
+]
+urls = { Homepage = "https://github.com/hhursev/recipe-scrapers/" }
+keywords = ["python", "recipes", "scraper", "harvest", "recipe-scraper", "recipe-scrapers"]
+license = { file = "LICENSE" }
```

Comment:
These four fields have slightly different output in `PKG-INFO`, as produced by `setuptools` v65.5.0, between the `setup.py` approach in the `main` branch and the `pyproject.toml` approach in this development branch.

- `Author` and `Author-email` fields are condensed into a single `Author-email` line (`Firstname Lastname <username@example.org>` format)
- Instead of a single `home-page` URL entry, a list of `Project-URL` entries with associated name, URL pairs is emitted (`Homepage, ...` in this case)
- Keywords are joined into a comma-separated list
- Instead of a `license-files` entry containing filenames, the license file's content is read and included into the `PKG-INFO` file

Those all seem fairly reasonable to me.